### ISSUE TITLE
sys-apps/ubuntu-app-launch-0.9_p20160928_p0_p01 is missing libertine.h

### ISSUE THREAD

**Issue by and3k** [shiznix/unity-gentoo#162]
Hi,

when I try to update ubuntu-app-launch, I get the build error:

```
[ 18%] Building CXX object libubuntu-app-launch/CMakeFiles/ubuntu-launcher.dir/application-impl-libertine.cpp.o
/var/tmp/portage/sys-apps/ubuntu-app-launch-0.9_p20160928_p0_p01/work/libubuntu-app-launch/application-impl-libertine.cpp:21:23: fatal error: libertine.h: No such file or directory
```

However, I also can’t install libertine:
```
$ emerge -av1 sys-apps/libertine                                                                                                                                                                                                                    

These are the packages that would be merged, in order:

Calculating dependencies... done!


[nomerge       ] sys-apps/libertine-1.4.2_p20161003_p0_p01::unity-gentoo  PYTHON_TARGETS="python3_4"
[ebuild  N    *]  x11-libs/content-hub-0.2_p20160914_p0_p01::unity-gentoo  0 KiB
[ebuild  N    *]   sys-apps/libertine-1.4.2_p20161003_p0_p01::unity-gentoo  PYTHON_TARGETS="python3_4" 0 KiB

Total: 2 packages (2 new), Size of downloads: 0 KiB

 * Error: circular dependencies:

(x11-libs/content-hub-0.2_p20160914_p0_p01:0/0::unity-gentoo, ebuild scheduled for merge) depends on
 (sys-apps/libertine-1.4.2_p20161003_p0_p01:0/0::unity-gentoo, ebuild scheduled for merge) (buildtime)
  (x11-libs/content-hub-0.2_p20160914_p0_p01:0/0::unity-gentoo, ebuild scheduled for merge) (buildtime)

 * Note that circular dependencies can often be avoided by temporarily
 * disabling USE flags that trigger optional dependencies.
```

Thanks,
Bela

**Comment by shiznix** [comment]
The circular dep. should be fixed in commit 8b81451de56fb876f1d03f6aae1518c270367356

Have added sys-apps/libertine to DEPEND in sys-apps/ubuntu-app-launch and so should be fixed in commit f0d579d382083282be7c31b1bba50b3d3d1238d0

Thanks :)

**Comment by and3k** [comment]
Great. Works! Thanks!
